### ISSUE TITLE
Issue with the age column

### ISSUE THREAD

**Issue by marvinford** [DSEI21000-S21/project-tennis-ml#4]
@vanomahi I don't know if this is a major issue, but I want to bring it to your attention.  I noticed that if a player played for 10 consecutive years on the WTA or ATP tour. The age for that player is different throughout the years, which is obvious. However, do we want to reflect that in the data or show the player's current age, irrespective of the past year's tournament that the player played?

**Comment by vanomahi** [comment]
> @vanomahi I don't know if this is a major issue, but I want to bring it to your attention. I noticed that if a player played for 10 consecutive years on the WTA or ATP tour. The age for that player is different throughout the years, which is obvious. However, do we want to reflect that in the data or show the player's current age, irrespective of the past year's tournament that the player played?

Right! Well, we could possibly just show the player's current age.
